scripts/test-turbo-modules.sh

```diff
@@ -269,10 +298,11 @@ check_lines() {
   check_line_unchanged "./android/cpp-adapter.cpp" "nativeMultiply"
   check_line_unchanged "./android/cpp-adapter.cpp" "::multiply"
 
-  check_line_unchanged "./ios/*.h" "#import"
-  check_line_unchanged "./ios/*.h" "Spec.h"
+  # check_line_unchanged "./ios/*.h" "#import"
+  # check_line_unchanged "./ios/*.h" "Spec.h"
+  check_line_unchanged "./ios/*.h" "Spec>"
   check_line_unchanged "./ios/*.h" "<Native"
-  check_line_unchanged "./ios/*.h" "<RCTBridgeModule"
+  # check_line_unchanged "./ios/*.h" "<RCTBridgeModule"
```

Comment:
If we're planning to uncomment these again later, a comment explaining why they're currently commented might help. Otherwise, I guess we could also just delete them?